app/_hub/kong-inc/grpc-web/index.md

```diff
@@ -0,0 +1,158 @@
+---
+name: gRPC-Web
+publisher: Kong Inc.
+
+categories:
+  - transformations
+
+type: plugin
+
+desc: Allow browser clients to call gRPC services.
+description: |
+  A Kong plugin to allow access to a gRPC service via the gRPC-Web protocol.
+  Primarily, this means JS browser apps using the gRPC-Web library.
+  
+source_url: https://github.com/Kong/kong-plugin-grpc-web
+
+license_type: MIT
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.0.x
+      - 1.5.x
+      - 1.4.x
+
+
+params:
+  name: grpc-web
+  route_id: true
+  protocols: ["http", "https"]
+  dbless_compatible: true
+
+  config:
+    - name: proto
+      required: false
+      default:
+      value_in_examples: path/to/hello.proto
+      description: |
+        If present, describes the gRPC types and methods.
+        Required to support payload transcoding.  When absent, the
+        web client must use application/grpw-web+proto content.
+
+---
+
+## Purpose
+
+A service that presents a gRPC API can be used by clients written in many languages, but the network specifications are oriented primarily to connections within a datacenter. [gRPC-Web] allows exposing such API to the Internet, and be consumed by brower-based JS apps.
```

Comment:
```suggestion
A service that presents a gRPC API can be used by clients written in many languages, but the network specifications are oriented primarily to connections within a datacenter. [gRPC-Web] allows exposing such API to the Internet so that it can be consumed by browser-based JS apps.
```

I was going to suggest for the typo alone, but I ended up tweaking the sentence too :)

Reply:
```suggestion
A service that presents a gRPC API can be used by clients written in many languages, but the network specifications are oriented primarily to connections within a datacenter. [gRPC-Web] lets you expose this API to the Internet so that it can be consumed by browser-based JS apps.
```
Slight further tweak to @hishamhm 's suggestion.